lib/info.js

```diff
@@ -426,6 +426,20 @@ const getM3U8 = async(url, options) => {
   return formats;
 };
 
+// Check for updates.
+const { version } = require('../package.json');
+let lastUpdateCheck = Date.now();
+const checkForUpdates = () => {
+  if (!process.env.YTDL_NO_UPDATE && Date.now() - lastUpdateCheck >= 1000 * 60 * 60 * 12) {
+    miniget('https://api.github.com/repos/fent/node-ytdl-core/releases/latest', { headers: { 'User-Agent': 'a/b' } })
```

Comment:
i was looking at github's docs for their api
https://developer.github.com/v3/#user-agent-required

let's put "ytdl-core" here for the User-Agent. they're hosting the repo after all.

Reply:
this fails the linter bc the line's too long. what do you suggest i do?

Reply:
add line breaks between the options
```js
..., {
  headers: { 'User-Agent': 'ytdl-core' },
})
```